client/app/states/designer/blueprints/editor/editor.html

```diff
@@ -1,15 +1,14 @@
-<span ng-controller="BlueprintEditorController" ng-class="{'wait-cursor': loading}">
+<span ng-controller="BlueprintEditorController as evm" ng-class="{'wait-cursor': loading}">
```

Comment:
I'd welcome any feedback in that regard.  The "Editor" controls the page level buttons/actions, while the "Canvas" controls the node CRUD; they are very much intertwined.  My only thought would be to make them two directives (?)